metrics_test.go

```diff
@@ -49,14 +53,18 @@ func TestClientIMPL_GetCapacity(t *testing.T) {
 }
 
 func TestClientIMPL_GetCapacity_Zero(t *testing.T) {
-	totalSpace := 1905262588
-	usedSpace := 12077448036352
+	totalSpace0 := 12077448036352
+	usedSpace0 := 10077448036352
+	totalSpace1 := 12077448036352
+	usedSpace1 := 11077448036352
+	totalSpace2 := 12077448036352
+	usedSpace2 := 12077448036353
 	var freeSpace int64
 
 	httpmock.Activate()
 	defer httpmock.DeactivateAndReset()
-	respData := fmt.Sprintf(`[{"physical_total": %d, "physical_used": %d}]`,
-		totalSpace, usedSpace)
+	respData := fmt.Sprintf(`[{"physical_total": %d, "physical_used": %d},{"physical_total": %d, "physical_used": %d},{"physical_total": %d, "physical_used": %d}]`,
+		totalSpace0, usedSpace0, totalSpace1, usedSpace1, totalSpace2, usedSpace2)
```

Comment:
Are we not checking for any output from the respData?

Reply:
This test is for zero capacity check (usedSpace2 > toalSpace2), GetCapacity() will return 0 in this case.